### PR TITLE
Update olcesium, ol, cesium

### LINKED ISSUE #3856: 3D b3dm  gLTF warnings in Building
open 
https://map.geo.admin.ch/?bgLayer=ch.swisstopo.pixelkarte-farbe&lang=fr&topic=ech&lon=7.14025&lat=46.61838&elevation=1063&heading=0.186&pitch=-88.418&layers=WMTS%7C%7Casitvd.fond_pourortho%7C%7Chttps:%2F%2Fows.asitvd.ch%2Fwmts%2F1.0.0%2FWMTSCapabilities.xml%3FSERVICE%3DWMS%26REQUEST%3DGetCapabilities%26VERSION%3D1.3.0

in console: following errors appear
image

Shoudl be fixed for acceptance protocol

### PR DESCRIPTION
**Pull request by oterral**
OL has added a `type` property to the layers classes, so we can't rely on it anymore.

Changes affects :
   - KML import (test if the Permalink is well updated)
   - WMS import  (test if the Permalink is well updated)
   - WMTS import (test if the Permalink is well updated)
   - updateDelay parameter use 

[Test](https://mf-geoadmin3.int.bgdi.ch/teo_up/index.html?topic=ech&bgLayer=ch.swisstopo.pixelkarte-farbe&layers=ch.swisstopo.zeitreihen,ch.bfs.gebaeude_wohnungs_register,ch.bav.haltestellen-oev,ch.swisstopo.swisstlm3d-wanderwege&layers_visibility=false,false,false,false&layers_timestamp=18641231,,,&lang=fr)

Fix #3856

### CODE REVIEW

**Comment by oterral** [comment]
Waitinfg for https://github.com/openlayers/ol-cesium/pull/485

**Comment by gjn** [comment]
@oterral ol3-cesium has been merged. Can you please integrate it here?

Once a new version is available, please inform @Sgachet @nadine-piveteau @rebert and me about it with an email. Extensive testing is needed - one most devices. Would be good to merge this today.

File: src/components/layermanager/LayermanagerDirective.js [hunk 1]
```diff
@@ -57,7 +57,7 @@ goog.require('ga_window_service');
     // Timestamps list template
     var tpl =
       '<div class="ga-layer-timestamps">' +
-        '<div tabindex="1" ng-if="tmpLayer.type == \'wms\'" ' +
+        '<div tabindex="1" ng-if="tmpLayer.timeBehaviour == \'all\'" ' +
```
Comment (gjn) [inline src/components/layermanager/LayermanagerDirective.js]: not sure if this is really equivalent.

Comment (oterral) [reply src/components/layermanager/LayermanagerDirective.js]: I've verified in the LayersConfig . All the layer with `timeBehaviour='all'`  are wms

**Comment by rebert** [comment]
@oterral When a clicking on your own drawing and hoovering over the icons of the to tooltip, the link on the buttom of the tooltip is hoovering as well. Tested on our nightly firefox.

BR
Tobias

**Comment by gjn** [comment]
Thanks @nadine-piveteau @Sgachet and @rebert for testing. And @oterral for the effort.

**Pull request merged**